app/models/user.rb

```diff
@@ -647,8 +646,128 @@ def banned_at_date?(date)
     groups
   end
 
+  def self.panel_pages
+    [
+      :postingDashboard,
+      :editPerson,
+      :regionsManager,
+      :groupsManagerAdmin,
+      :bannedCompetitors,
+      :translators,
+      :duesExport,
+      :countryBands,
+      :delegateProbations,
+      :xeroUsers,
+      :duesRedirect,
+      :delegateForms,
+      :regions,
+      :subordinateDelegateClaims,
+      :subordinateUpcomingCompetitions,
+      :leaderForms,
+      :groupsManager,
+      :importantLinks,
+      :delegateHandbook,
+      :seniorDelegatesList,
+      :leadersAdmin,
+      :boardEditor,
+      :officersEditor,
+      :regionsAdmin,
+      :downloadVoters,
+      :generateDbToken,
+    ].to_h { |panel_page| [panel_page, panel_page.to_s.underscore.dasherize] }
```

Comment:
Instead of this manual `to_h` approach, you can also use `index_with` or `index_by`. Both take a lambda function with _one_ parameter, and the difference is that:
- `index_with` turns an array into the hash using the original value as the key and the result of the lambda function as the new value
- and `index_by` uses the original value as the hash value and the result of the lambda function as the new key

https://apidock.com/rails/Enumerable/index_with
https://apidock.com/rails/Enumerable/index_by

Reply:
Thanks a lot, done.